packages/react-components/react-avatar/MIGRATION-AvatarGroup.md

```diff
@@ -6,30 +6,31 @@
 
 ## Migration from v8
 
-AvatarGroup and Facepile have similar APIs, but the main difference is how they interact with AvatarGroupItem/Persona. AvatarGroup receives AvatarGroupItems as children and places them in their respective place, this means AvatarGroup has no control over AvatarGroupItem. `size` is the only property of AvatarGroupItem that is modified by AvatarGroup to make all AvatarGroupItems the same size. Face adding functionality is not supported.
+AvatarGroup and Facepile have similar APIs, but the main difference is how they interact with AvatarGroupItem/Persona. `size` is the only property of AvatarGroupItem that is modified by AvatarGroup to make all AvatarGroupItems the same size. Face adding functionality is not supported.
```

Comment:
What is "Face adding functionality"?

Reply:
Facepile has two props, `addButtonProps` and `showAddButton`, that allow you to render a button at the beginning that adds personas when clicked. This is something built into Facepile. I'll remove it since the props mapping shows that it's not supported anyways.